src/script.c

```diff
@@ -141,6 +143,11 @@ void environment_exit(environment_t *env) {
 }
 
 bool execute_script(const char *name, environment_t *env) {
+	if(!enable_scripts) {
+		logger(DEBUG_ALWAYS, LOG_NOTICE, "Not running script %s as scripts are disabled", name);
```

Comment:
We shouldn't warn about this, if `enable_scripts` is `false`, it's because the user explicitly disabled it.